app/Models/Extensions/ThrowsConsistentExceptions.php

```diff
@@ -138,9 +138,10 @@ public function toJson($options = 0): string
 			// that this method does so.
 			// Hence, we call `json_encode` _without_ specifying
 			// `JSON_THROW_ON_ERROR` and then mimic that behaviour.
-			$json = json_encode($this->jsonSerialize(), $options);
-			if (json_last_error()) {
-				throw new \JsonException(json_last_error_msg(), json_last_error());
+			// ! TODO VERIFY THIS !
+			$json = \Safe\json_encode($this->jsonSerialize(), $options);
```

Comment:
```suggestion
			// TODO: Verify this!
			$json = \Safe\json_encode($this->jsonSerialize(), $options);
```

I have a déjà vu. There must be a similar comment somewhere above.